packages/vscode-extension/src/debug/npmInstallHandler.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as fs from "fs-extra";
+import * as path from "path";
+import * as vscode from "vscode";
+
+import { loadPackageJson } from "./commonUtils";
+
+export async function hasNpmInstalled(folder: string): Promise<boolean> {
```

Comment:
`isNpmInstalled` may be more accurate.

Reply:
Changed to `checkDependencies` to not confuse with *npm is installed*